integrationTests/multiShard/endOfEpoch/rating/rating_test.go

```diff
@@ -0,0 +1,656 @@
+package rating
```

Comment:
there is no test here - like no assert / require. it is hard to evaluate.

Reply:
Indeed, since this is not a test, rather than a tool, I suggest to properly make it a binary (contents contained in a main.go, program could have parameter inputs rather hardcoded values and so on). This will be useful in case we want to run this on a machine for days. Suggestion to have this aither moved inside elrond-go-tools repo or moved out of the integrationTests package.